tt_metal/impl/device/device.hpp

```diff
@@ -102,68 +93,54 @@ class Device {
 
     bool is_initialized() const { return this->initialized_; }
 
+    // Next group methods is proxying a call to Cluster
+    // Some methods can be replaced with ::get_soc_description
+    // dram_size_perf_channel is only used in tests
+    // dram_grid_size is only used in a test and reshard
```

Comment:
dram_grid_size is exposed at the python level and is a required function for dram sharding. This is used by multiple models.